src/pylogctx/celery.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from celery.app import current_task
+from celery.app.task import Task
+
+from .core import context
+
+
+class LoggingTask(Task):
+    def __call__(self, *args, **kwargs):
+        task = current_task()
+        context.update(
+            celeryTask=task.name,
+            celeryTaskId=task.request.id,
```

Comment:
Also, other stack (java, js, php, etc.) are also camel case, so, being camelcase is consistent. I guess that this is to save some space in column views (app_object_id > appObjectId).